splinepy/helpme/integrate.py

```
@@ -258,3 +308,7 @@ def volume(self, *args, **kwargs):
     @_wraps(parametric_function)
     def parametric_function(self, *args, **kwargs):
         return parametric_function(self._helpee, *args, **kwargs)
+
+    @_wraps(physical_function)
+    def physical_function(self, *args, **kwargs):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)